func_adl_xAOD/common/statement.py

```diff
@@ -65,24 +67,31 @@ def set_rep(self, name: Any, value: Any):
 class loop(block):
     'A for loop'
 
-    def __init__(self, loop_var_rep: crep.cpp_value, collection_rep: crep.cpp_collection):
+    def __init__(self, loop_var_rep: crep.cpp_value,
+                 collection_rep: crep.cpp_collection,
+                 is_loop_var_a_pntr=False, is_loop_var_a_ref=False):
```

Comment:
Should both these be default - or should we *always* specify this. If we should always specify these, then lets leave off the default values and add to the code. In short: the point isn't to not disturbe the old code, but make sure it is "right" for now.

Reply:
If the loop variable is a pointer then imo it will not be a reference and vice versa hence the default values.